src/main/java/org/humancellatlas/ingest/messaging/Constants.java

```diff
@@ -10,8 +10,10 @@
     public class Queues {
         public static final String BUNDLEABLE_PROCESS_SUBMITTED = "ingest.state-tracking.bundle.submitted.queue";
         public static final String BUNDLEABLE_PROCESS_COMPLETED = "ingest.state-tracking.bundle.completed.queue";
-        public static final String DOCUMENT_CREATED = "ingest.state-tracking.metadata.created.queue";
-        public static final String DOCUMENT_UPDATE = "ingest.state-tracking.metadata.update.queue";
+        //not used
+        //public static final String DOCUMENT_CREATED = "ingest.state-tracking.metadata.created.queue";
+        //probably also not used? commented queue config as well
+        //public static final String DOCUMENT_UPDATE = "ingest.state-tracking.metadata.update.queue";
```

Comment:
just delete code rather than commenting it out.
Can we can the team to confirm your assertion that there aren't used?